parts/linux/cloud-init/artifacts/cse_config.sh

```diff
@@ -478,6 +478,13 @@ configAzurePolicyAddon() {
     sed -i "s|<resourceId>|/subscriptions/$SUBSCRIPTION_ID/resourceGroups/$RESOURCE_GROUP|g" $AZURE_POLICY_ADDON_FILE
 }
 
+{{- if not IsAKSCustomCloud}}
+ensureTimesyncd() {
+    systemctlDisableAndStop chrony
+    systemctlEnableAndStart systemd-timesyncd
```

Comment:
do we need this?

Reply:
So after chrony is installed, it will be started automatically. I'm not 100% sure if that will affect systemd-timesyncd, it would be safer to just restart systemd-timesyncd.

Reply:
Actually we should probably do this at the very beginning, since it potentially affects the timestamp we print in logs.

Reply:
agree we should do it at the beginning.

Reply:
moved disabling to VHD